plugins/out_s3/s3_store.h

```diff
@@ -24,20 +24,22 @@
 #include <fluent-bit/flb_fstore.h>
 
 struct s3_file {
-    int locked;                      /* locked chunk is busy, cannot write to it */
+    int locked;                      /* locked = no appends to this chunk */
```

Comment:
Should just remove the unlock function, to prevent accidental bugs, its not needed anymore.